src/reducers/targetReducer.js

```diff
@@ -19,6 +28,22 @@ export default function targetReducer(state = initialState.targets, action){
       return newState;
     }
 
+    case types.LOAD_TARGETS:{
+      newState = action.targets;
+      return newState.map(el => {
+        return {
+          id: el.id,
```

Comment:
These params are the same that you used above. Maybe you could make a function to return these params in order of don't repeat code. Just a suggestion